Source/Fuse.Navigation/RoutePageProxy.uno

```diff
@@ -65,21 +65,23 @@ namespace Fuse.Navigation
 		}
 		List<Level> _levels = new List<Level>();
 		
+		/** `Init()` must be called to finish initialization. */
 		public RoutePageProxy( Visual source, ProgressUpdated progressUpdated )
 		{
 			_progressUpdated = progressUpdated;
-			Init(source);
+			_source = source;
 		}
 		
+		/** `Init()` must be called to finish initialization. */
 		public RoutePageProxy( Visual source, ActiveChanged activeChanged )
 		{
 			_activeChanged = activeChanged;
-			Init(source);
+			_source = source;
 		}
-		
-		void Init( Visual source )
+
+		/** Call after configuration is completed. This could start the actual registration and messaging */
+		public void Init()
```

Comment:
I thought about another approach and this one is more likely to not be fail. It's definitely an API failure in the way it was setup before. Making a live update to a `Path` change take effect would have been complicated, since it would already be in the middle of updating subscriptiosn. So this was the more obvious solution.

Since this API was first introduced in 1.1 though, and already broken, I think we're safe in changing it in the patch. Nobody could be using it effectively.